linode/manager
# feat: [M3-7169, M3-7193, M3-7211, M3-7255, M3-7258] - VPC UX feedback

## Description 📝
Address UX feedback for VPC

## Changes  🔄
- Updated VPC empty landing text
- New VPC icon
- Minor UI changes to Assign Linode to Subnet drawer
- Updated tooltip text for the Auto-assign a VPC IPv4 address checkbox
- Minor text UI changes to VPC details page

## Preview 📷
| VPC empty landing before  | VPC empty landing after   |
| ------- | ------- |
| ![image](https://github.com/linode/manager/assets/115299789/d1de3d7f-7b41-4a4d-acd2-864720f5e50e) | ![vpc empty after](https://github.com/linode/manager/assets/115299789/d40f04f9-7e47-44ef-a931-f26fdc06e91e) |

| VPC side/create menu icon before  | VPC side/create menu after   |
| ------- | ------- |
| ![image](https://github.com/linode/manager/assets/115299789/bdf0b5f1-3608-4b52-9ce2-767dd227be2f) | ![vpc icon after](https://github.com/linode/manager/assets/115299789/e9bc6621-1a75-49f4-96f7-6ab4e983f929) |

| Assign Linode to Subnet drawer before  | Assign Linode to Subnet drawer after   |
| ------- | ------- |
| ![Screenshot 2023-10-23 at 5 26 22 PM](https://github.com/linode/manager/assets/115299789/0efa411c-4c0d-4e83-b3e1-054bdd42e738) |  ![Screenshot 2023-10-23 at 5 27 24 PM](https://github.com/linode/manager/assets/115299789/c3cc6584-77cd-4e9e-bd48-a83d0540374b) |

| VPC detail Safari before  | VPC detail Safari after   |
| ------- | ------- |
| ![image](https://github.com/linode/manager/assets/115299789/3d0fdc32-c297-4f5f-800d-f8d10545af8c) | ![image](https://github.com/linode/manager/assets/115299789/7921ff31-9098-4f71-a7ea-a957bf5e7027) |


## How to test 🧪

### Prerequisites
(How to setup test environment)
- Point to the dev env and ensure your account has vpc customer tags

### Verification steps 
Verify the ticket changes requested and check these flows:
- VPC empty landing (set line 87 to true in `VPCLanding`)
- VPC side menu and create menu icons
- Assign Linode to Subnet drawer
- VPC details page (changes are more noticeable in Safari)

## As an Author I have considered 🤔

*Check all that apply*
